examples/.gitignore

```diff
@@ -1,4 +1,5 @@
 whylogs-output/
+whylogs-image/
```

Comment:
Is this required as part of the transport_parameters changes?

Reply:
i dont think this change requires a separate issue/PR and comes up during the testing of the notebooks locally rather annoying.

Reply:
Sure, no problem. Wanted to check it was intentional.